### ISSUE TITLE
Block Fortress

### ISSUE THREAD

**Issue by ShariQuinn** [Rinnegatamante/Android2Vita-Candidate-Ports-List#421]
### Submission Rules

- [X] I have read the README.md and understand the submission rules

### No Unity or Java Games

- [X] This game is NOT made in Unity and does not contain a libunity.so
- [X] This game is NOT made in Java and does not contain a libgdx.so

### No Duplicate Submissions

- [X] This game is NOT already in the Android2Vita candidate port list

### Game Name

Block Fortress

### Game Version

1.01.17

### ARM Build Version

ARMv7

### OpenGL Version

GLES 2

### FMOD Usage

No FMOD

### Listed or Delisted

Google Play Store

### Store Link

https://play.google.com/store/apps/details?id=com.foursakenmedia.blockfortress&hl=en&gl=US

**Comment by treasboi** [comment]
Wait this game isn't made from unity?

**Comment by ShariQuinn** [comment]
I don't believe so? I followed the rules guide and saw no unity files in the apk
